src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,89 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.math.BigDecimal;
+import java.util.List;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@SpringBootTest
+@Sql(value = "classpath:data/truncate.sql")
+class MenuServiceTest {
```

Comment:
서비스 테스트 구성 정보를 상속 구조로 가져가면 이후에 관리하기 좋을 수도 있을거 같아요!
개인적인 의견입니다.

```suggestion
class MenuServiceTest extends ServiceTestConfig {
```

Reply:
구현하면서 상속으로 뺄까? 하는 생각을 여덟번쯤 했던 것 같은데 역시 빼야겠습니다.

아무 메서드도 없는 클래스가 될 것 같아서 고민했었는데, 사실 저도 상속으로 구현하고 싶었어요 ㅎㅎ

Reply:
[7350c0f](https://github.com/woowacourse/jwp-refactoring/pull/490/commits/7350c0f49e26fbe13a2ff3e0d173a9d383efd466)